packages/nameguard-react/src/components/NameBadge/NameBadge.tsx

```diff
@@ -1,47 +1,89 @@
 import React, { type MouseEventHandler } from "react";
 import type { ConsolidatedNameGuardReport } from "@namehash/nameguard";
+import cc from "classcat";
 
 import { NameShield } from "../NameShield";
 import { Shield } from "../Report/Shield";
+import { ShieldError } from "../Shield/Error";
 
 type NameBadgeProps = {
   placeholder?: string;
   data?: ConsolidatedNameGuardReport;
   onClick?: MouseEventHandler<HTMLButtonElement>;
+  forceError?: boolean;
```

Comment:
I didn't want to add this but only did so that we could present the `error` state on the demo page. I'd prefer to remove this and do it properly, but we never discussed said additions to SDK.

Reply:
Btw, there are two types of possible errors here. An error from NG API and one from the request itself (offline).

I can remove the `forceError` for now and show the `ShieldError` component when the request returns an `error`. I just have no way to display that on this page currently.

Reply:
Removed `forceError`.